stacker_blueprints/dynamodb.py

```diff
@@ -0,0 +1,152 @@
+from stacker.blueprints.base import Blueprint
+from . import util
+
+from troposphere import (
+    dynamodb2,
+    Ref,
+    GetAtt,
+    Output,
+)
+
+from troposphere.route53 import RecordSetType
+
+def prep_schemata(config):
+    try:
+        schemata = []
+        for schema in config["KeySchema"]:
+            schemata.append(dynamodb2.KeySchema(**schema))
+
+    except IndexError:
```

Comment:
I think this should be `KeyError`, you're trying to catch `"KeySchema"` not being in `config` right?